openupgrade_scripts/scripts/hr_attendance/17.0.2.0/pre-migration.py

```diff
@@ -42,5 +69,6 @@ def fill_res_company_attendance_from_systray(env):
 @openupgrade.migrate()
 def migrate(env, version):
     openupgrade.rename_xmlids(env.cr, _xmlid_renames)
+    openupgrade.rename_fields(env, _fields_renames)
```

Comment:
Encapsulate this into a module explaining that this comes from `hr_attendance_geolocation`, and only being called if you find the SQL column.